scout/server/blueprints/variant/templates/variant/tx_overview.html

```diff
@@ -69,7 +69,7 @@
                   <td class="d-flex align-items-center">
                     <div> <!-- ID col-->
                       <span class="text"> {{ transcript.transcript_id }}</span>
-                      {% if gene.common and transcript.refseq_id in gene.common.primary_transcripts %}
+                      {% if (primary_transcripts in gene.common) and (gene.common and transcript.refseq_id in gene.common.primary_transcripts) %}
```

Comment:
```suggestion
                      {% if gene.common and 'primary_transcripts' in gene.common and transcript.refseq_id in gene.common.primary_transcripts %}
```

Reply:
Jinja is slight different from python and to check that a key exists in a dictionary you check it either with

`if 'key' in dict ` or with `if dict.key`

We've found this out with last week, remember @dnil ? 🤣

Reply:
😅 Yes, that was not totally obvious. Flask2 is like the Dragos, tough on the tough?

Reply:
https://www.naspbokhandel.se/pub_images/large/9789176217115_1.Jpeg?timestamp=1616910123

Reply:
@northwestwitch Ill try that solution.